app/tests/Commands/System/UninstallCommandTest.php

```diff
@@ -231,8 +231,8 @@ public function test_execute() : void
             ->with("doil")
         ;
 
-        $tester->setInputs(["y"]);
-        $execute_result = $tester->execute([]);
+        $tester->setInputs(["y", "y"]);
+        $execute_result = $tester->execute(["--prune" => true]);
```

Comment:
Maybe add a second test without the prune option?

Reply:
Done!